specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2023-03-01/PrometheusRuleGroups.json

```diff
@@ -0,0 +1,507 @@
+          },
+          "description": "Resource tags"
+        },
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "the flag that indicates whether the Prometheus rule group is enabled."
+            }
+          }
+        }
+      },
+      "description": "The Prometheus rule group resource for patch operations."
+    },
+    "PrometheusRuleGroupResourceCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrometheusRuleGroupResource"
+          },
+          "description": "the values for the alert rule resources."
+        }
+      },
+      "description": "Represents a collection of alert rule resources."
+    },
+    "PrometheusRuleGroupProperties": {
+      "description": "An Azure Prometheus rule group.",
+      "type": "object",
+      "required": [
+        "scopes",
+        "rules"
+      ],
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "Rule group description."
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable/disable rule group."
+        },
+        "clusterName": {
+          "type": "string",
+          "description": "Apply rule to data from a specific cluster."
+        },
+        "scopes": {
+          "$ref": "#/definitions/StringArray",
+          "description": "Target Azure Monitor workspaces resource ids. Only one scope currently supported."
+        },
+        "interval": {
+          "type": "string",
+          "description": "The interval in which to run the Prometheus rule group represented in ISO 8601 duration format. Should be between 1 and 15 minutes",
+          "format": "duration"
+        },
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrometheusRule"
+          },
+          "description": "Defines the rules in the Prometheus rule group.",
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "PrometheusRule": {
+      "type": "object",
+      "description": "An Azure Prometheus alerting or recording rule.",
+      "required": [
+        "expression"
+      ],
+      "properties": {
+        "record": {
+          "description": "Recorded metrics name.",
+          "type": "string"
+        },
+        "alert": {
+          "description": "Alert rule name.",
+          "type": "string"
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Enable/disable rule."
+        },
+        "expression": {
+          "description": "The PromQL expression to evaluate. Every evaluation cycle this is evaluated at the current time, and the result recorded as a new set of time series with the metric name as given by 'record'.",
```

Comment:
>evaluate [](http://example.com/codeflow?start=51&length=8)

nit, Consider documenting in more depth like supported language version, reference to language docs?